.github/workflows/dotnet-core-win.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        version: [net452,net46,net461,netcoreapp3.1]
+        version: [net452,net46,net461,netcoreapp2.1,netcoreapp3.1]
```

Comment:
netcore2.1 is already installed on win image?

Reply:
yes, it is